starters/gatsby-wordpress-starter/src/components/footer.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import { Link, useStaticQuery, graphql } from "gatsby"
+
+const Footer = () => {
+  const {
+    wpMenu: {
+      menuItems: { nodes },
+    },
+  } = useStaticQuery(graphql`
+    query MenuQuery {
+      wpMenu(name: { eq: "Example Menu" }) {
+        id
+        menuItems {
+          nodes {
+            id
+            path
+            label
+          }
+        }
+      }
+    }
+  `)
+
+  const FooterMenu = () => (
+    <nav className="flex flex-col max-w-lg mx-auto lg:max-w-screen-lg">
+      <ul>
+        {nodes.map(({ id, label, path }, i) => {
+          return (
+            <li
+              key={id + i}
+              className="ml-3 list-disc text-blue-300 hover:text-blue-100"
+            >
+              <Link
+                className="text-blue-300 hover:text-blue-100 focus:text-purple-600 active:text-purple-300 "
+                to={`${path}`}
```

Comment:
Since the posts don't have the prefix coming from the backend, let's hardcode it here
```suggestion
                to={`/posts${path}`}
```